etrago/tools/utilities.py

```diff
@@ -1079,6 +1089,8 @@ def delete_dispensable_ac_buses(etrago):
     None.
 
     """
+    if etrago.args["delete_dispensable_ac_buses"] is False:
+        return
```

Comment:
It is a bit strange that the function is called anyway. Why didn't you add the if-statement here?: https://github.com/openego/eTraGo/blob/a8515570275f68f3fcfd7a2de48becad0c689144/etrago/tools/network.py#L406

Reply:
I followed the implementation of the load_shedding, with the difference that I did not include everything inside an if statement.
https://github.com/openego/eTraGo/blob/a8515570275f68f3fcfd7a2de48becad0c689144/etrago/tools/utilities.py#L586C1-L604C35

Reply:
Sorry for the comment :)